CMakeLists.txt

```diff
@@ -107,15 +107,10 @@ option(ENABLE_TESTING "Whether to enable the unit testing build" ON)
 #Check if there is need to compile unit test
 if(ENABLE_TESTING)
     add_subdirectory(test)
-
     include(CTest)
-    
-    add_executable(tests test/TestMain.cpp)
-    find_package(Catch2 CONFIG REQUIRED)
-    target_link_libraries(tests PRIVATE Catch2::Catch2 Catch2::Catch2WithMain)
-    else()
-        message("W/o exe. unit")
-    endif()

```

Comment:
Are we ignoring the unit tests?

Reply:
I removed those lines because `tests/CMakeLists.txt` included by `add_subdirectory(test)` already adds an executable target for `test/TestMain.cpp`, so it seemed to be duplicate. I don't think adding back the lines should break anything, but I still don't believe they are needed.